src/print.js

```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+  console.log('I get called from print.js!');
+}
```

Comment:
- [ ] KIndly remove this file, it doesn't serve any purpose in this project also its bad practise to have console.log in your codebase.